risc0/zkp/Cargo.toml

```diff
@@ -25,8 +25,8 @@ risc0-core = { workspace = true }
 risc0-sys = { workspace = true, optional = true }
 risc0-zeroio = { workspace = true }
 risc0-zkvm-platform = { workspace = true }
-rustacuda_core = { version = "0.1", optional = true }
-rustacuda_derive = { version = "0.1", optional = true }
+rustacuda_core = { version = "0.1.2", optional = true }
+rustacuda_derive = { version = "0.1.2", optional = true }
```

Comment:
No need for this, if we leave off the 'build' part, it should be fine. See: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html